src/lib/sort/sort-header.scss

```diff
@@ -11,6 +11,7 @@ $mat-sort-header-arrow-hint-opacity: 0.38;
   display: flex;
   cursor: pointer;
   align-items: center;
+  outline: 0;
```

Comment:
Add a comment that explains why `outline` is hidden?

Reply:
Done.